### PR TITLE
intel-media-driver: new package at 20.2.0

### PR DESCRIPTION
**Pull request by mmatongo**
---

## Description of package

Intel Media Driver for VAAPI — Broadwell+ iGPUs
## Installed manifest of package

```
/var/db/kiss/installed/intel-media-driver/version
/var/db/kiss/installed/intel-media-driver/sources
/var/db/kiss/installed/intel-media-driver/manifest
/var/db/kiss/installed/intel-media-driver/depends
/var/db/kiss/installed/intel-media-driver/checksums
/var/db/kiss/installed/intel-media-driver/build
/var/db/kiss/installed/intel-media-driver/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/lib/pkgconfig/igfxcmrt.pc
/usr/lib/pkgconfig/
/usr/lib/libigfxcmrt.so.7.2.0
/usr/lib/libigfxcmrt.so.7
/usr/lib/libigfxcmrt.so
/usr/lib/libigdgmm.a
/usr/lib/dri/iHD_drv_video.so
/usr/lib/dri/
/usr/lib/
/usr/include/igfxcmrt/cm_rt_g9.h
/usr/include/igfxcmrt/cm_rt_g8.h
/usr/include/igfxcmrt/cm_rt_g12_tgl.h
/usr/include/igfxcmrt/cm_rt_g11.h
/usr/include/igfxcmrt/cm_rt_g10.h
/usr/include/igfxcmrt/cm_rt_extension.h
/usr/include/igfxcmrt/cm_rt_def_os.h
/usr/include/igfxcmrt/cm_rt_api_os.h
/usr/include/igfxcmrt/cm_rt.h
/usr/include/igfxcmrt/cm_hw_vebox_cmd_g10.h
/usr/include/igfxcmrt/
/usr/include/
/usr/
```

## New package

- [Y] Latest upstream version.
- [Y] I agree to maintain this package.
- [Y] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

**Comment by mmatongo** [comment]
I hadn't updated my fork in a while thats why I was getting conflicts, sorry about that.

File: community/intel-media-driver/build [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+sed -i 's/SHARED/STATIC/' gmmlib/Source/GmmLib/CMakeLists.txt
+
+cd media
+
+cmake -B build \
+		-DCMAKE_INSTALL_PREFIX=/usr \
+		-DCMAKE_INSTALL_LIBDIR=/usr/lib \
+        -DBS_DIR_GMMLIB="$OLDPWD/gmmlib/Source/GmmLib" \
+		-DINSTALL_DRIVER_SYSCONF=OFF \
+        -DRUN_TEST_SUITE=OFF
```
Comment (dylanaraps) [inline community/intel-media-driver/build:14]: Indentation is 4 spaces.

File: community/intel-media-driver/build [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+sed -i 's/SHARED/STATIC/' gmmlib/Source/GmmLib/CMakeLists.txt
+
+cd media
+
+cmake -B build \
+	-DCMAKE_INSTALL_PREFIX=/usr \
+	-DCMAKE_INSTALL_LIBDIR=/usr/lib \
+        -DBS_DIR_GMMLIB="$OLDPWD/gmmlib/Source/GmmLib" \
+	-DINSTALL_DRIVER_SYSCONF=OFF \
+        -DRUN_TEST_SUITE=OFF
```
Comment (dylanaraps) [inline community/intel-media-driver/build:14]: Indentation is still wrong. :+1:

Comment (mmatongo) [reply community/intel-media-driver/build]: ugh sorry, don't know what happened.

**Comment by mmatongo** [comment]
@dylanaraps anything wrong with this?

File: community/intel-media-driver/build [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+sed -i 's/SHARED/STATIC/' gmmlib/Source/GmmLib/CMakeLists.txt
+
+cd media
+
+cmake -B build \
+	-DCMAKE_INSTALL_PREFIX=/usr \
+	-DCMAKE_INSTALL_LIBDIR=/usr/lib \
+    -DBS_DIR_GMMLIB="$OLDPWD/gmmlib/Source/GmmLib" \
+	-DINSTALL_DRIVER_SYSCONF=OFF \
+    -DRUN_TEST_SUITE=OFF
```
Comment (dylanaraps) [inline community/intel-media-driver/build:14]: Indentation is still off (4 spaces).

Comment (mmatongo) [reply community/intel-media-driver/build:14]: This is odd, on my branch it's just fine.
[SEE](https://github.com/mmatongo/community/blob/intel-media-driver/community/intel-media-driver/build)

**Pull request merged**